tests/test_router_ospf_vrf.rb

```diff
@@ -196,6 +192,7 @@ def test_routerospfvrf_collection_not_empty_valid
       refute_empty(vrfs, 'Error: Collection is empty')
       hv = ospf_h.fetch(routername.to_s)
       next if hv.nil?
+      hv = hv['default']
```

Comment:
Needed because of the above refactoring. Previously `test_routerospfvrf_collection_not_empty_valid` had a hash of the form `ospf_h[ospf][property]` while `test_routerospfvrf_collection_router_multi_vrfs` had a hash of the form `ospf_h[ospf][vrf][property]`. When I refactored to let these two tests share the `config_from_hash` function, I had to add the extra `[vrf]` layer to this hash, even though this test only uses vrf `default`.